docs-ref-conceptual/spring-framework/configure-spring-boot-java-applicationinsights.md

```diff
@@ -0,0 +1,314 @@
+  <Properties>
+    <Property name="LOG_PATTERN">
+      %d{yyyy-MM-dd HH:mm:ss.SSS} %5p ${hostName} --- [%15.15t] %-40.40c{1.} : %m%n%ex
+    </Property>
+  </Properties>
+  <Appenders>
+    <Console name="Console" target="SYSTEM_OUT">
+      <PatternLayout pattern="%d{HH:mm:ss.SSS} [%t] %-5level %logger{36} - %msg%n"/>
+    </Console>
+    <ApplicationInsightsAppender name="aiAppender">
+    </ApplicationInsightsAppender>
+  </Appenders>
+  <Loggers>
+    <Root level="trace">
+      <AppenderRef ref="Console"  />
+      <AppenderRef ref="aiAppender"  />
+    </Root>
+  </Loggers>
+</Configuration>
+```
+4. Build and run the spring boot application again as shown above. 
+
+Within few seconds, you should see all the spring logs being avaialble on Azure Portal. 
+
+![Azure Portal][AZ06]
+
+You can even look at the detailed log messages and do analysis on Analytics Portal. 
+
+![Azure Portal][AZ07]
+
+## Next steps
+
+For more information about using Spring Boot applications on Azure, see the following articles:
+
+* [Deploy a Spring Boot Application to the Azure App Service](deploy-spring-boot-java-web-app-on-azure.md)
+
+* [Running a Spring Boot Application on a Kubernetes Cluster in the Azure Container Service](deploy-spring-boot-java-app-on-kubernetes.md)
+
+Application Insights supports automatic collection of external dependencies and it's correlation with incoming requests. Currently we support autocollection of Oracle, MsSQL, MySQL and Redis. For more details on enabling autocollection please follow the article [how to use Application Insights Java agent](https://docs.microsoft.com/en-us/azure/application-insights/app-insights-java-agent).
+
+For more information about Azure Application Insights, and it's monitoring capabilities, see the **[Application Insights]** home page.
+
+For more information about additional configuration details of Application Insights Spring Boot Starter, please refer to this [link](https://github.com/Microsoft/ApplicationInsights-Java/blob/master/azure-application-insights-spring-boot-starter/README.md).
+
+For feature requests and potential bugs, please open issues on our [GitHub](https://github.com/Microsoft/ApplicationInsights-Java/issues) repository.
+
+For more information about using Azure with Java, see the [Azure for Java Developers] and the [Java Tools for Visual Studio Team Services].
+
+The **[Spring Framework]** is an open-source solution that helps Java developers create enterprise-level applications. One of the more-popular projects that is built on top of that platform is [Spring Boot], which provides a simplified approach for creating stand-alone Java applications. To help developers get started with Spring Boot, several sample Spring Boot packages are available at <https://github.com/spring-guides/>. In addition to choosing from the list of basic Spring Boot projects, the **[Spring Initializr]** helps developers get started with creating custom Spring Boot applications.
```

Comment:
<https://github.com/spring-guides/> with just angle brackets may not render and will likely get skipped over when the doc publishes. I would recommend including the link this way with square bracket+angle bracket notation as you do with your other links in the doc:

```[https://github.com/spring-guides/](https://github.com/spring-guides/)```